addons/account/wizard/account_payment_register.py

```diff
@@ -526,31 +528,35 @@ def _compute_partner_bank_id(self):
     @api.depends('payment_type', 'journal_id', 'currency_id')
     def _compute_payment_method_line_fields(self):
         for wizard in self:
-            if wizard.journal_id:
-                wizard.available_payment_method_line_ids = wizard.journal_id._get_available_payment_method_lines(wizard.payment_type)
-            else:
-                wizard.available_payment_method_line_ids = False
+            wizard.available_payment_method_line_ids = wizard._get_available_payment_method_lines(wizard.payment_type)
```

Comment:
It's strange that the code is different for the same computation on `account.payment`